src/actions/sources/tests/select.spec.js

```diff
@@ -89,16 +89,20 @@ describe("sources", () => {
   it("should select next tab on tab closed if no previous tab", async () => {
     const { dispatch, getState } = createStore(sourceThreadClient);
 
-    const fooSource = makeSource("foo.js");
+    const bazSource = makeSource("baz.js");
+
     await dispatch(actions.newSource(fooSource));
```

Comment:
argh.. this should be `await dispatch(actions.newSource(makeSource("foo.js")));`